### ISSUE TITLE
Cannot send_keys russian letters

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#136]
```
This script:

retCode, inp = dialog.input_dialog(title="title", message="message", 
default="default")
keyboard.send_keys(inp)

Works fine for english letters, but try to input russian letters, e.g. this 
line: привет. Nothing will be inserted (tested in GNU Emacs).
```

Original issue reported on code.google.com by `kheyfbo...@gmail.com` on 12 Aug 2011 at 7:20

**Comment by GoogleCodeExporter** [comment]
```
Thanks for this report. How are your keyboard layouts set up? Do you use 
multiple layouts and switch between them?
```

Original comment by `cdekter` on 11 Sep 2011 at 7:13

**Comment by GoogleCodeExporter** [comment]
```
Yes exactly that: I use two layouts and switch between them.
```

Original comment by `kheyfbo...@gmail.com` on 12 Sep 2011 at 11:02

**Comment by GoogleCodeExporter** [comment]
```
Therein lies the problem - AutoKey does not support layout switching at all and 
will get very confused. Try setting a single layout, rebooting and then seeing 
if it still is a problem.
```

Original comment by `cdekter` on 12 Sep 2011 at 10:09

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in upcoming release 0.80.0
```

Original comment by `cdekter` on 18 Sep 2011 at 11:27
- Changed state: **Fixed**
